ethereum/contracts/ETHAtomicSwapper.sol

```diff
@@ -0,0 +1,219 @@
+pragma solidity ^0.5.8;
+
+contract ETHAtomicSwapper {
```

Comment:
what's the difference between this and the above one

Reply:
ERC20 swap contract and ETH swap contract have some difference in transfer asset.